ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/validation/ErrorFactoriesSpec.scala

```diff
@@ -79,6 +79,133 @@ class ErrorFactoriesSpec
       )
     }
 
+    "TrackerErrors" should {
+
+      val errorDetails = com.google.protobuf.Any.pack[ErrorInfo](
+        ErrorInfo
+          .newBuilder()
+          .build()
+      )
```

Comment:
I think it might be possible to remove this empty errorDetails from the assertions if you just build the statuses directly in v1 (mentioned in previous comment)

Reply:
I think these `errorDetails` actually existed in the original code since I took care to preserve the way the original code was building these errors.
I believe there are appearing where a io.grpc.Status is converted to rich Status.

I'm a bit reluctant to change these tests now as:
1) this is what original code did
2) these tests will get deleted together with all V1 errors (?)soon.

Let me know if you insist though then I'll change it :)